amplitude-segment-snippet.min.js

```diff
@@ -1,3 +1,6 @@
-(function(e,t){var n=e.amplitude||{};n._q=[];function o(e){n[e]=function(){n._q.push([e].concat(Array.prototype.slice.call(arguments,0)));
-}}var r=["init","logEvent","logRevenue","setUserId","setUserProperties","setOptOut","setVersionName","setDomain","setDeviceId","setGlobalUserProperties"];
-for(var s=0;s<r.length;s++){o(r[s])}e.amplitude=n})(window,document);
+(function(t,e){var n=t.amplitude||{};n.Identify=function(){this.p={};Object.keys(identifyFuncs).forEach(function(t){
```

Comment:
Somehow I don't think the segment snippet minimize generated properly. I don't see where the identifyFuncs are being declared